apps/api/src/modules/stores/stores.service.spec.ts

```diff
@@ -1,20 +1,129 @@
+import { CreateStoreDto, UpdateStoreDto } from '@meal-time/api-interfaces';
 import { ConfigService } from '@nestjs/config';
 import { Test, TestingModule } from '@nestjs/testing';
 import { PrismaService } from '../../services/prisma/prisma.service';
 import { StoresService } from './stores.service';
 
 describe('StoresService', () => {
-  let service: StoresService;
+	let module: TestingModule;
+	let service: StoresService;
+	let prisma: PrismaService;
+	beforeAll(async () => {
+		module = await Test.createTestingModule({
+			providers: [StoresService, PrismaService, ConfigService],
+		}).compile();
 
-  beforeEach(async () => {
-    const module: TestingModule = await Test.createTestingModule({
-      providers: [StoresService, PrismaService, ConfigService],
-    }).compile();
+		service = module.get<StoresService>(StoresService);
+		prisma = module.get<PrismaService>(PrismaService);
+	});
 
-    service = module.get<StoresService>(StoresService);
-  });
+	beforeEach(async () => {
+		await prisma.cleanDb();
+	});
 
-  it('should be defined', () => {
-    expect(service).toBeDefined();
-  });
+	it('should create store', async () => {
+		const dto: CreateStoreDto = {
+			name: 'Meny',
+		};
+		const store = await service.create(dto);
+
+		if (store) {
+			return expect(store.name).toEqual('Meny');
+		}
+	});
+
+	it('should find all stores', async () => {
+		const dto: CreateStoreDto = {
+			name: 'Brugsen',
+		};
+		const dto2: CreateStoreDto = {
+			name: 'Fakta',
+		};
+		const store1 = await prisma.stores.create({
+			data: {
+				...dto,
+			},
+		});
+		const store2 = await prisma.stores.create({
+			data: {
+				...dto2,
+			},
```

Comment:
It is totally fine to use the other functions of the service even if it is not the one you are testing. This will reduce the complexity of the tests and won't impact the tests as long as all functions are covered in their own tests

In this case it would be
```ts
const store1 = await service.create(dto);
const store2 = await service.create(dto2);
```